src/database/migrations/20221004193118_create_user_table.ts

```diff
@@ -0,0 +1,21 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('user', (table) => {
+    table.increments('id');
+    table
+      .string('email', 254) // https://www.rfc-editor.org/errata_search.php?rfc=3696
+      .notNullable();
+    table.string('forename', 255).notNullable();
+    table.string('surname', 255).notNullable();
+    table.boolean('admin').notNullable();
+    table.boolean('blender').notNullable();
+    table.string('salt', 128).notNullable();
+    table.string('password_hash', 255).notNullable();
+    table.timestamps(true, true);
+  });
+}
+
+export async function down(knex: Knex): Promise<void> {
+  await knex.schema.dropTable('user');
```

Comment:
Imo implementing this is not necessary, since if we later on want to delete user table for some reason then we will create another migration to handle it. But there is no harm on having it either, so no need to remove it

Reply:
I have used it about 10 times already :D